packages/core/review-workflows/server/src/services/stages.ts

```diff
@@ -210,7 +210,7 @@ export default ({ strapi }: { strapi: Core.Strapi }) => {
       const entity = await strapi.documents(model).update({
         documentId,
         locale,
-        data: { [ENTITY_STAGE_ATTRIBUTE]: stage },
+        data: { [ENTITY_STAGE_ATTRIBUTE]: pick(['id'], stage) },
```

Comment:
when connecting a relation with an "id" , it skips the document id+locale+status transformation. As the review workflow stage doesnt have i18n nor DP, its ok to skip it

Reply:
maybe this could be a comment in the code?

Reply:
yes!! good point
just added a comment